automated install/basic-install.sh

```diff
@@ -2229,7 +2363,7 @@ FTLinstall() {
     printf "  %b %s..." "${INFO}" "${str}"
 
     # Find the latest version tag for FTL
-    latesttag=$(curl -sI https://github.com/pi-hole/FTL/releases/latest | grep "Location" | awk -F '/' '{print $NF}')
+    latesttag=$(curl -sI https://github.com/pi-hole/FTL/releases/latest | grep "location" | awk -F '/' '{print $NF}')
```

Comment:
This is different that what is in master. Master already has `grep -i` to account for capitalization.

https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh#L2387

Reply:
okay, I'm trying to figure out how to resolve the conflict.  I did 'git fetch upstream' and my copy of development still does not have grep -i.